packer/builder/azure/arm/step_delete_os_disk.go

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See the LICENSE file in the project root for license information.
+
+package arm
+
+import (
+	"fmt"
+	"net/url"
+	"strings"
+
+	"github.com/Azure/packer-azure/packer/builder/azure/common/constants"
+
+	"github.com/mitchellh/multistep"
+	"github.com/mitchellh/packer/packer"
+)
+
+type StepDeleteOSDisk struct {
```

Comment:
Do we need this explicitly? Doesn't deleting the VM (by deleting the RG) also delete the OS VHD blob?